### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by fungs** [biothings/mygene.py#7]
In the download packages, mygene/__init__.py contains the actual code. In the github master repo, this is empty. Is this software maintained?

**Comment by newgene** [comment]
@fungs that's because we now have a unified [biothings_client.py](https://github.com/biothings/biothings_client.py) for all of our [BioThings APIs](http://biothings.io).

With biothings_client, you can get mygene client like this:

```
from biothings_client import get_client
mg = get_client('gene')
```

Then the rest is the same as the current mygene.py client.

We will still keep "mygene.py" client, and it will become a simple wrapper around "biothings_client.py". From the user side, it will be exactly the same:

```
import mygene
mg = mygene.MyGeneInfo()
```

We should make a release for the new "mygene" python package. If you want to check the current v3.0.0 code, you can always have it at its "v3.0.0" tag:

https://github.com/biothings/mygene.py/tree/ver_3.0.0

**Comment by newgene** [comment]
Just a note for anybody visited this issue:

The released `mygene` v3.1.0 package is now a thin wrapper around `biothings_client` package as described above. But the use of `mygene` package is exactly the same. More details at:

https://pypi.org/project/mygene/3.1.0/
and
https://mygene-py.readthedocs.io/en/latest/
